do-release.sh

```diff
@@ -158,11 +172,10 @@ This is a minor release.
 ### External Contributions
 
 EOF
-echo
-echo "Press enter to continue..."
-read
+
 git commit -a -m "Prepare next development version"
 git push origin ${CURRENT_BRANCH}
+./mvwn -B release:clean
```

Comment:
I guess, that's about pom.xml.release files cleaning up?

Reply:
the backup poms actually, but yes.